FAQ.md

```diff
@@ -176,92 +176,93 @@ Post quality is judged by the amount of effort you put into the creation of your
+### Q: How do I sell items in the shop?
+**A**: You need to contact the admins and let them know what you want to sell. We will help you make a post and set the price.
 
-### Q: How to remove my post from the forum? I can't see any option.
-**A**: First you need to click on the report button, this can be found at the bottom of the post. Then you need to give the report reason and click on the report. Alternatively, under the [Telegram channel](https://t.me/joinchat/KUVnAlQmTohLLwZEZcGv8A) drop your link and ask for a remove. In both ways, you have to wait until the staff process your report.
+### Q: How do I remove my post from the forum? I can't see any option.
+**A**: First you need to click on the report button, this can be found at the bottom of the post. Then you need to give the report reason and click on the report. Alternatively, under the [Telegram channel](https://t.me/joinchat/KUVnAlQmTohLLwZEZcGv8A), drop your link and ask for it to be remove. In both ways, you have to wait until the staff process your report.
 
-### Q: Approval applies to all sections?
-**A**: If you have a BOOMER tag, then only Golden Guide section posts need approval. However, if you don't have a BOOMER Tag, then all your post need approval. 
+### Q: Does approval apply to all sections?
+**A**: If you have the Boomer tag, then only posts in the [Golden Guides](https://blackpearl.biz/forums/154/) section need approval. However, if you don't have the Boomer tag, then all your posts need approval. 
 
 ### Q: How can I change my user name?
-**A**: If you have a DONATOR Tag, then you can change your user name in every 24 hours. Moreover, If you do not have a DONATOR Tag, then you can change your user name only once in 30 days.
+**A**: If you have a Donator tag, then you can change your user name every 24 hours. Moreover, if you do not have a Donator tag, then you can change your user name only once every 30 days.
 
-### Q: Can we buy reaction score for pearls? If no, why not?
-**A**: No, you can not buy reaction score pearls. Because it will result in spamming in the forum.
+### Q: Can we buy reaction score with pearls? If no, why not?
+**A**: No, you can not buy reaction score with pearls as it would result with spamming in the forum.
 
-### Q: Whom to contact for which issues?
-**A**: For any account related issues, Contact ADMINS (members whose user ID is in RED)
-       For Post related issues, Contact section Moderators.
+### Q: Who should I contact for which issues?
+**A**: For any Account related issues, Contact ADMINS (members whose user ID is in RED)
+       For Post related issues, Contact that section's Moderators.
 
-### Q: what about alternative accounts?
-**A**: No alternate accounts are allowed. If found any, both accounts will be banned.
+### Q: Are alternate accounts allowed?
+**A**: No, alternate accounts are NOT allowed. If any are found, both accounts will be banned.
```

Comment:
''are found'' > ''were found''

Reply:
For this I'm going to have to use "were to be found" to keep the future tense of the sentence intact. otherwise it would be mixing past tense and future tense in the same sentence. let me know if that's okay for you or if you think it should be something different.